docs/index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   argus:
   - apiVersion: v2
-    created: "2020-10-05T15:18:25.480623+02:00"
+    created: "2020-10-06T00:35:58.533822+02:00"
```

Comment:
I believe there are no manual edits for this file. Sometimes helm changes digests for unchanged charts too - not sure if helm refers some kind of temporal variable for that - but as long as index.yaml is created by helm, that is ok.

Reply:
Exactly, That was weird for me as well. Why does it changes the other charts.